includes/admin/class-sensei-extensions.php

```diff
@@ -113,6 +113,25 @@ public function get_extensions( $type = null, $category = null, $additional_quer
 		return $extensions;
 	}
 
+	/**
+	 * Get Sensei extensions and WooCommerce.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param  string $type                  Product type ('plugin' or 'theme').
+	 * @param  string $category              Category to fetch (null = all).
+	 * @param  string $additional_query_args Additional query arguments.
+	 * @return array
+	 */
+	public function get_extensions_and_woocommerce( $type = null, $category = null, $additional_query_args = [] ) {
```

Comment:
It basically extends the method `get_extensions` and includes the `woocommerce`. We could include the woocommerce directly to the `get_extensions`, but I didn't think it would make sense since it's not a Sensei extension.